src/rpa_gw.F

```diff
@@ -4777,13 +4778,14 @@ SUBROUTINE get_sigma_c_bisection_pade(gw_energ, Eigenval_scf, Sigma_x_minus_vxc_
                                      coeff_pade, sigma_c)
          qp_energy = Eigenval_scf + REAL(sigma_c) + Sigma_x_minus_vxc_gw
          delta = qp_energy - qp_energy_old
-         IF (icount > 500) THEN
-            WRITE (n_level_gw_ref_char, '(I10)') n_level_gw_ref
-            WRITE (error_msg, '(A,A,A)') " Self-consistent quasi-particle solution of "// &
-               "MO ", TRIM(n_level_gw_ref_char), " has not been found."
-            CPWARN(error_msg)
-            EXIT
-         END IF
+!         IF (icount > 500) THEN
+!            WRITE (n_level_gw_ref_char, '(I10)') n_level_gw_ref
+!            WRITE (error_msg, '(A,A,A)') " Self-consistent quasi-particle solution of "// &
+!               "MO ", TRIM(n_level_gw_ref_char), " has not been found."
+!            CPWARN(error_msg)
+!            EXIT
+!         END IF
```

Comment:
Wrap it in IF (.FALSE.) THEN ... END IF or remove it (same below)

Reply:
sure!